src/index.js

```diff
@@ -387,7 +387,7 @@ function getItemValues(item, key) {
     value = item[key]
   }
   // concat because `value` can be a string or an array
-  return value ? [].concat(value) : null
+  return value || value === 0 ? [].concat(value) : null
```

Comment:
I'd rather change this to:

```js
return value == null ? [].concat(value) : null
```

That way we cover null and undefined and all other cases will be concatenate.